homeworks/Aleksandr.Turuta_turuta7/homework_7/homework.test.js

```diff
@@ -0,0 +1,19 @@
+const { cycleAndTest } = require('./homework_7');
+
+describe('test fibonacci', () => {
+    test('test fibonacci', () => {
```

Comment:
`test()` method string argument should tell what are you actually testing about your unit now, e.g. `test('it should alway return an array'...` or `test('the first argument of returned array should be 0'...`

such way you and other developers will see what tests go OK and what not